test/test_openmp.py

```diff
@@ -0,0 +1,94 @@
+import collections
+import sys
+import time
+import unittest
+
+import numpy as np
+import torch
+from torch.testing._internal.common_utils import (
+    TestCase, run_tests, TEST_WITH_ASAN)
+
+try:
+    import psutil
+    HAS_PSUTIL = True
+except ImportError:
+    HAS_PSUTIL = False
+
+device = torch.device('cpu')
+
+
+# Use the C function rather than trying to set the environment variable
```

Comment:
let's update and move the comment to make it more clear, I guess you meant torch.set_num_threads

Reply:
I removed the comment altogether in a094af4 and turned other comments into docstrings. This comment was a left-over from the first version of the test, since then I added numerous calls to `torch.set_num_threads` so the environment setting probably would not have worked anyway.